Podfile

```diff
@@ -12,6 +12,7 @@ pod 'NSURL+IDN', '0.3'
 pod 'DTCoreText',   '1.6.13'
 pod 'UIDeviceIdentifier', '~> 0.1'
 pod 'SVProgressHUD', '~>1.1.3'
+pod 'SDWebImage', '~>3.6'
```

Comment:
Just a side note, if we are starting to use this library for image caching maybe add some issues on Github to replace other caches mechanism we are using. Or even a post on CoreApps explaining the advantages of using SDWebImage.